### ISSUE TITLE
Replace copyright and license in all source and header files

### ISSUE THREAD

**Issue by garyo** [AcademySoftwareFoundation/openfx#108]
we should probably change the copyright in the header files now that OpenFX is officially an Academy Software Foundation project. I'm happy to do that (using a script). We are using the BSD 3-Clause License. 

I assume we should use a modern SPDX-License-Identifier comment block; perhaps something like this:

```c
// Copyright (c) Contributors to the OpenFX project.
// SPDX-License-Identifier: BSD-3-Clause
// https://github.com/AcademySoftwareFoundation/OpenFx
```

Is that sufficient or should we add the license text itself, a copyright year, or anything else? Is the copyright attribution correct? (I got it from OpenShadingLanguage and OpenColorIO's header is similar.)

By the way, I notice a few `//` style comments in the headers although we mostly avoid them. I presume we are OK with this now, since `//` comments were added to ANSI C in 1999. Let me know if you have any problem with that.

**Comment by revisionfx** [comment]
New file can bear new (c)

The theory from our transition research for this, as it would have otherwise required we get a signature from all who may have contributed (and their successors for those who are not alive)
Old files we edit should have 2 lines, (c) OpenFX 2003-2022
(c) new one  2022-23

**Comment by garyo** [comment]
Now being implemented in #110
